### ISSUE TITLE
Split configuration into 'App Settings' and 'Build sezzigns' 

### ISSUE THREAD

**Issue by Mijago** [Mijago/D2ArmorPicker#123]
Split the (advanced) configuration into two chunks:
- App Settings - everything that does not directly affect a build. These must be consistent over reloads and 'reset all'. 
- build settings + everything that affects which and how armor/Mods are selected. 

The benefit of this is that we will be able to introduce more settings that only impact the app and not the builds, thus reducing the size of exported builds.

**Comment by maxchristy** [comment]
So, when you say "These must be consistent over reloads and 'reset all'.", you're intending for the App settings to be persistent when clicking the red "Clear everything" button?

![image](https://user-images.githubusercontent.com/47064300/236309393-3fa9f8a2-57a8-49d0-8603-b16f6aa1daad.png)

**Comment by TheMojoDojo** [comment]
The App settings should be consistent over the "Clear Everything" button, as you've said. 

The "Reset Everything" button located on the account tab should probably clear out the account settings in addition to the build settings as well. Sort of as a factory reset in worst case scenarios

**Comment by Mijago** [comment]
Yes. App Settings should be everything that is not build related, such as performance and other settings that may come up.

The idea is that we do not share settings that may impact performance or change settings that we otherwise wouldn't ever touch.

**Comment by maxchristy** [comment]
Do we want a button to reset Account settings without having to reset the whole application?
